src/ContainerIntegration.jsx

```diff
@@ -12,7 +12,7 @@ const _ = cockpit.gettext;
 
 // ports is a mapping like { "5000/tcp": [{"HostIp": "", "HostPort": "6000"}] }
 export const renderContainerPublishedPorts = ports => {
-    if (!ports)
+    if (Object.keys(ports).length === 0)
         return null;
```

Comment:
Good catch. Keep this in a separate commit when consolidating the commits :D

Reply:
We generally don't squash PR's and merge this as `rebase & merge`